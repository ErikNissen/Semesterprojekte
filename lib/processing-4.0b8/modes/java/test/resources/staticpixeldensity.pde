version https://git-lfs.github.com/spec/v1
oid sha256:da08c86d8667b0ea3cfee532d16528fe93c8866d75556eb69739b6ced70b1be6
size 110
