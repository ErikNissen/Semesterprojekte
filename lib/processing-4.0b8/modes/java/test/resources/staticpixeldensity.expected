version https://git-lfs.github.com/spec/v1
oid sha256:1b7aafb3ec94111b9e4c40f425cd76b038e59c616a1edad05178a9e4e5d986f3
size 965
